docs/source/API/core/initialize_finalize/initialize.rst

```diff
@@ -0,0 +1,91 @@
+``initialize``
+==============
+
+.. role::cpp(code)
+    :language: cpp
+
+Defined in header ``<Kokkos_Core.hpp>``
+
+Usage: 
+
+.. code-block:: cpp
+
+    Kokkos::initialize(argc, argv);
+    Kokkos::initialize(Kokkos::InitializationSettings()  // (since 3.7)
+                           .set_disable_warnings(true)
+                           .set_num_threads(8)
+                           .set_map_device_id_by("random"));
+
```

Comment:
Is the code above intended to be pseudo code?  If it is live, working, copy-paste-able code, forgive my inexperience. I ask, because I would expect working (copy-paste-able) code in these blocs.  In other words:

```
Kokkos::initialize(argc, argv);
Kokkos::initialize(Kokkos::InitilizationSettings().set_disable_warnings(true),
                             Kokkos::InitilizationSettings().set_num_threads(8),
                             Kokkos::InitilizationSettings().set_map_device_id_by("random"));

```

Reply:
it is not pseudo code, it works because it is a chain of things